docs/A-Introduction/information.md

```diff
@@ -1,5 +1,9 @@
 ---
+id: information
 sidebar_position: 2
+title: Information
+slug: /A-Introduction/information
```

Comment:
Hey, great work on this Md file,
Something that I noticed was:

Slug should be standardized in small caps and without the "A-"
Check https://github.com/Seneca-ICTOER/IPC144/issues/64 for more details.

Thank you.

Reply:
thx, I have modified.